src/clj/game/cards/programs.clj

```diff
@@ -2785,6 +2785,55 @@
                                                  (trigger-event state :corp :no-action))}])
                                 (make-run eid :archives (get-card state card)))}]})
 
+(defcard "Sneakdoor Prime A"
+         {:abilities [{:cost [:click 2]
```

Comment:
Indentation

Reply:
really need to figure out how to get IntelliJ to indent clj properly

Reply:
There should be some way to mark `defcard` as `def`, so it'll just inherit those indentation settings.

Reply:
https://cursive-ide.com/userguide/formatting.html#code-style-settings look there

Reply:
Ah awesome, that totally did the trick! Classic JetBrains "command+return" gets you 99% of the way to anything.